prerequirements.txt

```diff
@@ -1 +1,2 @@
 gdal==1.10.0
+protobuf==3.1.0
```

Comment:
Thanks for the explanation @kotfic I am assuming that before it was getting protobuf somehow (the newest one). Nonetheless, the changes looks good to me.